pkg/platformconfig/platformconfig.go

```diff
@@ -186,6 +190,52 @@ func (c *Config) GetFunctionMetricSinks() (map[string]MetricSink, error) {
 	return c.getMetricSinks(c.Metrics.Functions)
 }
 
+func (c *Config) EnrichContainerResources(ctx context.Context,
+	logger logger.Logger,
+	resources *v1.ResourceRequirements) {
+
+	defaultFunctionPodResources := c.Kube.DefaultFunctionPodResources
+
+	logger.DebugWithCtx(ctx,
+		"Populating resources with default values",
+		"defaultFunctionPodResources", defaultFunctionPodResources)
+
+	if resources.Requests == nil {
+		resources.Requests = make(v1.ResourceList)
+	}
+
+	if resources.Requests["cpu"] == (apiresource.Quantity{}) {
```

Comment:
what's that condition is about? what are you checking?

Reply:
It's a silly way of doing this:
```
if _, exists := resources.Requests["cpu"]; !exists {
    // enrich
}
```

I'll fix it